src/_posts/platform/deployment/2000-01-01-deploy-with-gitlab.md

```diff
@@ -61,6 +61,13 @@ With "Auto deploy" enabled, your application will be deployed every time you
 push to a selected branch of your repository. You can enable the "Auto deploy"
 on your application when creating the link or later, with the following command:
 
+{% note %}
```

Comment:
Can you also move the note elsewhere and not after the following paragraph:
```
With "Auto deploy" enabled, your application will be deployed every time you
push to a selected branch of your repository. You can enable the "Auto deploy"	push to a selected branch of your repository. You can enable the "Auto deploy"
on your application when creating the link or later, with the following command:	on your application when creating the link or later, with the following command:
```

Reply:
Moving it :slightly_smiling_face: